platform/darwin/src/NSDate+MGLAdditions.h

```diff
@@ -7,4 +7,7 @@
 /// Converts from a duration in seconds to a duration object usable in mbgl.
 mbgl::Duration MGLDurationInSeconds(NSTimeInterval duration);
 
+/// Converts from an mbgl duration object to a duration in seconds.
+NSTimeInterval MGLSecondsFromDuration(mbgl::Duration duration);
```

Comment:
`MGLDurationInSecondsFromTimeInterval()`, actually. `mbgl::Duration` can be expressed in many units; seconds is the one we’ve chosen to use for consistency with `NSTimeInterval`, but other parts of mbgl use milliseconds instead.